### DISCUSSION TITLE
Programmatically Connect to WiFi Network as in AirBnb app

### DISCUSSION THREAD

**Discussion by theobouwman** [expo/expo#19052] [Archived: Questions]
In the AirBnb app it is possible to connect to the WiFi network of the booking by pressing a button.

How can I achieve this?

**Comment by keith-kurak** [comment]
Hi! This functionality does not exist in the Expo SDK itself, but, you may find a 3rd party React Native module that does provide this functionality. For features like this, it can help to look for the native feature provided by Apple or Google (e.g., [NEHotspotConfigurationManager](https://developer.apple.com/documentation/networkextension/nehotspotconfigurationmanager)), and then search if a package has been created that exposes that native API to React Native. If it doesn't exist yet, it may be something you could create with the help of Expo's [modules API](https://docs.expo.dev/modules/overview/#use-the-expo-module-api-in-an).
